crux-http-server/src/crux/http_server/util.clj

```diff
@@ -59,12 +79,58 @@
            (transit/writer output-stream :json options) data)
           (.flush output-stream))))))
 
+(defn crux-stringify-keywords
+  [m]
+  (let [f (fn [[k v]]
+            (cond
+              (= :crux.db/id k) ["_id" v]
+              (= :crux.db/fn k) ["_fn" (pr-str v)]
+              (keyword? k) [(name k) v]
+              :else [k v]))
+        g (fn [k] (cond
+                    (= :crux.db/id k) "_id"
+                    (keyword? k) (name k)
+                    :else k))]
+    (walk/prewalk (fn [x]
+                     (cond
+                       (map? x) (into {} (map f x))
+                       (vector? x) (mapv g x)
+                       :else x))
+                   m)))
+
+(defn crux-object-mapper [{:keys [camel-case? mapper-options] :as options}]
```

Comment:
think this will indiscriminately convert keys to camelCase, even within users' documents?